components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -437,6 +437,7 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
     } else if ([keyPath isEqualToString:kTitlePositionAdjustment]) {
       itemView.titlePositionAdjustment = [change[kNewString] UIOffsetValue];
     }
+    [itemView setSelected:itemView.selected animated:NO];
```

Comment:
Does this line need to be called for every KVO or can we add it to only those paths that require an update?

Reply:
Originally I only added it to image, and selected image changes but then remembered that title presentation also needs configuration. I chose to do it this way because I don't have that much knowledge of what exactly needs to be configured. If you can give guidance as to which specific changes need configuration I'll go ahead and update the PR accordingly.

Reply:
Is the title presentation a separate issue? The PR is only mentioning the image color at the moment.